File: app/src/androidTest/java/de/christinecoenen/code/zapp/app/mediathek/MediathekData.java
package de.christinecoenen.code.zapp.app.mediathek;

import de.christinecoenen.code.zapp.app.mediathek.model.MediathekShow;

public class MediathekData {

	public static MediathekShow getTestShow() {
		MediathekShow mediathekShow = new MediathekShow();
		mediathekShow.setApiId("test123");
		mediathekShow.setChannel("ARD");
		mediathekShow.setTitle("My Title");
		mediathekShow.setVideoUrl("http://download.blender.org/peach/bigbuckbunny_movies/BigBuckBunny_320x180.mp4");
		return mediathekShow;
	}

}

File: app/src/androidTest/java/de/christinecoenen/code/zapp/app/mediathek/MediathekData.kt
package de.christinecoenen.code.zapp.app.mediathek

import de.christinecoenen.code.zapp.models.shows.MediathekShow

object MediathekData {

	val testShow: MediathekShow
		get() = MediathekShow(
			apiId = "test123",
			channel = "ARD",
			title = "My Title",
			videoUrl = "http://download.blender.org/peach/bigbuckbunny_movies/BigBuckBunny_320x180.mp4"
		)

}
